src/config.rs

```diff
@@ -0,0 +1,207 @@
+use std::env;
+use std::fs;
+use std::io::{Read, Write};
+use std::path::PathBuf;
+
+use ansi_term::{Colour, Style};
+use toml;
+use xdg::BaseDirectories;
+
+use error::TealdeerError::{self, ConfigError};
+
+pub const SYNTAX_CONFIG_FILE_NAME: &'static str = "syntax.toml";
+
+#[derive(Copy, Clone, Debug, Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord)]
+pub enum RawColour {
+    Black,
+    Red,
+    Green,
+    Yellow,
+    Blue,
+    Purple,
+    Cyan,
+    White,
+}
+
+impl From<RawColour> for Colour {
+    fn from(raw_colour: RawColour) -> Colour {
+        match raw_colour {
+            RawColour::Black => Colour::Black,
+            RawColour::Red => Colour::Red,
+            RawColour::Green => Colour::Green,
+            RawColour::Yellow => Colour::Yellow,
+            RawColour::Blue => Colour::Blue,
+            RawColour::Purple => Colour::Purple,
+            RawColour::Cyan => Colour::Cyan,
+            RawColour::White => Colour::White,
+        }
+    }
+}
+
+#[derive(Copy, Clone, Debug, Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
True, usually I lean towards more derives than less, because third party users, even within a crate might have use cases that depend on some derives, however here there is no user of ord, plus this type does not get exposed. I'll remove them.